serve/mlc_serve/engine/sampling_params.py

```diff
@@ -62,21 +65,27 @@ class SamplingParams:
     top_p: float = 1.0
     top_k: int = -1
     logit_bias: Optional[Dict[int, float]] = None
-    appeared_tokens_freq: Dict[int, int] = None
     logit_bias_index: list[int] = None
     logit_bias_value: list[float] = None
     logprobs: bool = False
     top_logprobs: int = 0
+    # TODO(@team): This info comes from the model config.
+    # Currently, it is unclear what is the best way to fetch this info and
+    # check in `_verify_args` without this field. Follow-up when we have a better idea.
+    vocab_size = 32000
```

Comment:
Add a ticket about this, this seems like a sharp edge that we are likely to run into.